test/ts-node-unit-tests/benchmarks/Stock/Stock-DataGrouping-Ordinal.bench.ts

```diff
@@ -0,0 +1,88 @@
+import type { BenchmarkContext, BenchmarkResult } from '../../benchmark';
+import { performance } from 'node:perf_hooks';
+import { join } from 'node:path';
+
+
+export const config = {
+    sizes: [1000, 10_000, 100_000]
+};
+
+export function generateOHLC(size: number){
```

Comment:
Shall we move it to a separate file, with data generators? Instead of other tests depending on this test